singleton.bundle.js

```diff
@@ -2,6 +2,8 @@ function singleton(factory) {
     let instance = null;
     return {
         getInstance: ()=>instance == null ? instance = factory() : instance
+        ,
```

Comment:
?

Reply:
Man the deno bundler is just outright ridiculous